src/styles/table.scss

```diff
@@ -0,0 +1,24 @@
+.table {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  width: 944px;
```

Comment:
It would be great to calculate this value, not hardcore it
![image](https://user-images.githubusercontent.com/96409563/228145896-11a8fffd-b6fa-4aec-a4db-ac271a9cb72f.png)